data/releases/schedule.yaml

```diff
@@ -14,11 +17,14 @@ schedules:
       targetDate: 2022-05-24
 - release: 1.23
   releaseDate: 2021-12-07
-  next: 1.23.9
-  cherryPickDeadline: 2022-07-08
-  targetDate: 2022-07-13
+  next: 1.23.10
+  cherryPickDeadline: 2022-08-12
+  targetDate: 2022-08-17
   endOfLifeDate: 2023-02-28
   previousPatches:
+    - release: 1.23.9
```

Comment:
```suggestion
    - release: 1.23.10
```

Reply:
since the last patch release has been 1.23.9, this should be 1.23.9 only, right?